handbook/repository/system-e2e.md

```diff
@@ -84,6 +96,8 @@ Copy to `apps/system-e2e/src/fixtures/<your-app>/users.json` and modify:
 ]
 ```
 
+Or you could try out `yarn playwright generate island.is`... We haven't QA'd it yet 🥰
```

Comment:
You probably mean this:

```suggestion
Or you could try out `yarn playwright codegen island.is`... We haven't QA'd it yet 🥰
```

Not sure about recommending stuff we haven't tried 😅 Maybe better to suggest it as a way to learn to write Playwright tests in an interactive way. But I'd warn users from copy pasting it directly, bad selectors and all.